pkg/validator/schema.go

```diff
@@ -124,6 +126,17 @@ func applyPodSchemaChecks(conf *config.Configuration, controller kube.GenericWor
 		if err != nil {
 			return nil, err
 		} else if check == nil {
+			check, err = resolveCheck(conf, checkID, controller, config.TargetController, false)
```

Comment:
I think this should go in its own function, `applyControllerSchemaChecks`, which could get called in `pkg/validator/controller.go`. That's similar to how we do both pod and container targets.

Reply:
Ohh yeah that makes a lot of sense, I was thinking of it still as controller checks, but it's still explicitly not a podSchemaCheck.